.github/workflows/updatenet-update.yml

```diff
@@ -0,0 +1,88 @@
+---
+name: UpdateNet Create
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: Name of deployment (updatenet)
+        required: true
+        type: string
+      destination:
+        description: Node image tag, commit sha short or "testnet" or "mainnet"
+        required: true
+        type: string
+        default: testnet
+
+jobs:
+  check-vars-and-secrets:
+    name: Check vars and secrets
+    uses: ./.github/workflows/_check-vars-and-secrets.yml
+    secrets: inherit
+
+  update-updatenet:
+    needs: [check-vars-and-secrets]
+    name: Update updatenet
+    runs-on: ubuntu-20.04
+    outputs:
+      deployment-id: ${{ steps.deployment.outputs.deployment_id }}
+    steps:
+      - name: Validate inputs
+        shell: bash
+        run: |
+          if [[ ! "${{ inputs.name }}" =~ ^[a-z0-9][a-z0-9\-]{4,30}$ ]]; then
+            echo "!!! Invalid updatenet name"
+            exit 1
+          fi
+          if [[
+            "${{ inputs.destination }}" != "" && \
+            ! "${{ inputs.destination }}" =~ ^[a-f0-9]{7}$
+          ]]
+          then
+            echo "!!! Invalid destination node image tag"
+            exit 1
+          fi
+
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Start featurenet Deployment
+        uses: bobheadxi/deployments@v1.1.0
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.CI_GH_TOKEN }}
+          env: updatenet-${{ inputs.name }}
+          override: true
+
+      - name: Create featurenet data, app and deploy it
+        uses: ./.github/actions/create-featurenet
```

Comment:
Does it rewrite whole FR from scratch, including wiping DB? Then unfortunately we need to rework it, this workflow should only do following on the given update net:
1. Stop old binary
2. Start a new binary  
nothing more

Reply:
Actually it was happening.  However, the condition wasn't quite clear.  It's fixed now and there's an explicit `update` input to `create-featurenet` action.